src/app/organizations/manage/collections.component.ts

```diff
@@ -147,10 +171,28 @@ export class CollectionsComponent implements OnInit {
         });
     }
 
+    async resetPaging() {
+        this.pagedCollections = [];
+        this.loadMore();
+    }
+
+    isSearching() {
+        return this.searchService.isSearchable(this.searchText);
+    }
+
+    isPaging() {
+        const searching = this.isSearching();
+        if (searching && this.didScroll) {
+            this.resetPaging();
+        }
+        return !searching && this.collections.length > this.pageSize;
+    }
+
     private removeCollection(collection: CollectionView) {
         const index = this.collections.indexOf(collection);
         if (index > -1) {
             this.collections.splice(index, 1);
+            this.resetPaging();
```

Comment:
Would we want to completely reset paging to the beginning when a collection is deleted? Does this reset the view in that sense so if I scroll through 5 pages, delete an item, I have to scroll back through those same 5 pages again to get back to where I was at?

Reply:
Negative - the `resetPaging` is basically used to catch up the paging list with the master list. Deleting a collection near the end of your list, has the same visible affect as deleting a collection at the top of your list.